buildscripts/condarecipe.local/meta.yaml

```diff
@@ -32,15 +32,15 @@ requirements:
     - numpy
     - setuptools
     # On channel https://anaconda.org/numba/
-    - llvmlite >=0.33.0dev0,<0.34
+    - llvmlite >=0.34.0dev0,<0.35
```

Comment:
Whilst similar previous patches, I'm not convinced by this, I think it's saying "anything from tag 0.34.0dev0" i.e. when work started for 0.34, "through to 0.35 final". I think it might need to say `>=0.35.0dev0` so that it included only development work from the 0.35 series.

Reply:
Right, i think `>=0.35.0dev0` will be needed